theano/tensor/signal/tests/test_conv.py

```diff
@@ -82,14 +82,13 @@ def sym_conv2d(input, filters):
 
     def test_basic(self):
         """
-        Basic functionality of nnet.conv.ConvOp is already tested by
-        its own test suite.  We just have to test whether or not
-        signal.conv.conv2d can support inputs and filters of type
-        matrix or tensor3.
+        Basic functionality of nnet.abstract_conv.AbstractConvOp is
+        already tested by its own test suite.  We just have to test
+        whether or not signal.conv.conv2d can support inputs and
+        filters of type matrix or tensor3.
         """
-        if(not theano.tensor.nnet.conv.imported_scipy_signal and
-           theano.config.cxx == ""):
-            raise SkipTest("conv2d tests need SciPy or a c++ compiler")
+        if(theano.config.cxx == "" or theano.config.mode == "FAST_COMPILE"):
+            raise SkipTest("conv2d tests need a c++ compiler")
```

Comment:
In FAST_COMPILE, LocalOptGroup would transform AbstractConv2d to CorrMM, which do not has perform method. It would raise MethodNotDefined error. Should I implement this method ?